captionsProcessor.py

```diff
@@ -0,0 +1,25 @@
+from config import Config
+from transcriptLoader import getCombinedTranscripts
+from topicExtractor import retrieveTopics
+from utils import printAndLog
+
+
+def main():
+    config = Config()
+    config.setFromEnv()
+
+    videoToUse = "New Google Assignments in Canvas"
```

Comment:
I would suggest this value as configurable value rather having it in code. You could make this a list, if there is a plan to run it multiple video caption at time.

Reply:
You are right, I mentioned this in the ReadMe as well, and never got to implementing it. 
I'll keep it as a single file for now. We could later add another script that does this over a list by calling this script? I think that will help to set up multiprocessing if we wanted that.